package.json

```diff
@@ -25,7 +25,7 @@
     "@angular/router": "10.0.8",
     "@highcharts/map-collection": "^1.1.3",
     "core-js": "^3.6.4",
-    "highcharts": "^8.1.2",
+    "highcharts": "^8.2.2",
```

Comment:
Seems like unrelated change.

Reply:
Yes, it doesn't change anything in the wrapper itself. 
This changes only the `higcharts` version in the demos which are attached to the repository.

Reply:
Maybe demo dependency update should be in a different PR?